models/experiment.py

```diff
@@ -31,6 +32,21 @@ class Experiment(OrmBase):
     studyId: Mapped[str] = mapped_column(ForeignKey('Study'), nullable=False)
     study: Mapped['Study'] = relationship(back_populates='experiments')
 
+    cultivationMode:    Mapped[str] = mapped_column(String(50))
+    controlDescription: Mapped[str] = mapped_column(String)
+
+    @hybrid_property
+    def id(self):
+        return self.experimentUniqueId
+
+    @hybrid_property
+    def name(self):
+        return self.experimentId
+
+    @hybrid_property
+    def description(self):
+        return self.experimentDescription
```

Comment:
At some point, I'll just make some migrations to rename fields and make them consistent, but in order to do this, I also need to fix the code that invokes them. For now, these aliases provide a simple transitional state.